crates/service/src/status.rs

```diff
@@ -30,6 +30,44 @@ impl<S: Service> ServiceMonitor<S> {
     pub fn into_generic(self) -> GenericStatusMonitor {
         GenericStatusMonitor::new(self)
     }
+
+    /// Creates a listener input for this service monitor.
+    ///
+    /// This allows another service to listen to status updates from this service.
+    /// Returns an MPSC receiver that will receive status updates, and spawns a task
+    /// to forward watch channel updates to it.
+    ///
+    /// The listener task will automatically exit when the monitored service exits.
+    pub fn create_listener_input(
+        &self,
+        executor: &strata_tasks::TaskExecutor,
+    ) -> crate::TokioMpscInput<S::Status> {
```

Comment:
Should import this at the top instead of using `crate::` here.

Reply:
done